ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.14__ocp-cnv-odf-4.14.yaml

```diff
@@ -0,0 +1,55 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  ocs-ci-tests:
+    name: ocs-ci-container
+    namespace: ci
+    tag: stable
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- context_dir: cnv-ci
+  dockerfile_path: Dockerfile
+  to: cnv-ci
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: cnv-odf-tests-aws-upi-ocp414
+  cron: 4 14 * * SAT
+  steps:
+    cluster_profile: aws-interop-qe
+    env:
+      BASE_DOMAIN: release-ci.cnv-qe.rhood.us
+      OCP_VERSION: "4.14"
+      ODF_OPERATOR_CHANNEL: stable-4.13
```

Comment:
ODF version is 4.13, is this ok?

Reply:
For 4.14 yes, for 4.15 I need to change it to use a ICSP to pull from the dev channel